src/main/java/main/internet/shop/controller/IndexController.java

```diff
@@ -31,8 +30,13 @@ public void init(ServletConfig config) throws ServletException {
         productService.create(new Product("sword", 34D));
         productService.create(new Product("spear", 45D));
         productService.create(new Product("shield", 22D));
-        userService.create(new User("test"));
-        shoppingCartService.create(new ShoppingCart(TEMP_USER_ID));
+        User admin = new User("admin", "admin", "123");
+        admin.addUserRole(UserRole.of("ADMIN"));
+        User userAdmin = new User("userAdmin", "userAdmin", "1234");
+        userAdmin.addUserRole(UserRole.of("ADMIN"));
+        userAdmin.addUserRole(UserRole.of("USER"));
+        userService.create(admin);
+        userService.create(userAdmin);
```

Comment:
let's create a separate InjectDataController for this purpose
we should not place business logic in the `init()` method